<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />



<meta name="date" content="2020-11-18" />

<title>Starting kit</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="site_libs/anchor-sections-1.0/anchor-sections.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cancer Heterogeneity</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="data_challenges.html">
    <span class="fa fa-universal-access"></span>
     
    Data Challenges
  </a>
</li>
<li>
  <a href="cometh.html">
    <span class="fa fa-cogs"></span>
     
    Cometh
  </a>
</li>
<li>
  <a href="deconbench.html">
    <span class="fa fa-code-fork"></span>
     
    Deconbench
  </a>
</li>
<li>
  <a href="https://cancer-heterogeneity.github.io/blog/">
    <span class="fa fa-rss"></span>
     
    Blog
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:magali.richard@univ-grenoble-alpes.fr, yuna.blum@ligue-cancer.net">
    <span class="fa fa-envelope-o"></span>
     
    Contact us
  </a>
</li>
<li>
  <a href="https://github.com/cancer-heterogeneity">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Starting kit</h1>
<h4 class="date">2020-11-18</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#overview-scientific-and-clinical-context"><span class="toc-section-number">1</span> Overview: Scientific and clinical context</a></li>
<li><a href="#computational-methods"><span class="toc-section-number">2</span> Computational methods</a><ul>
<li><a href="#the-different-types-of-deconvolution-methods"><span class="toc-section-number">2.1</span> The different types of deconvolution methods</a></li>
<li><a href="#criteria-used-for-the-selection"><span class="toc-section-number">2.2</span> Criteria used for the selection</a></li>
<li><a href="#selection-of-13-reference-methods"><span class="toc-section-number">2.3</span> Selection of 13 reference methods</a></li>
</ul></li>
<li><a href="#evaluation-of-methods-performances"><span class="toc-section-number">3</span> Evaluation of methods performances</a><ul>
<li><a href="#reference-datasets"><span class="toc-section-number">3.1</span> Reference datasets</a></li>
<li><a href="#scoring-metrics"><span class="toc-section-number">3.2</span> Scoring metrics</a></li>
<li><a href="#benchmark-leaderboard"><span class="toc-section-number">3.3</span> Benchmark leaderboard</a></li>
</ul></li>
<li><a href="#clinician-datasets"><span class="toc-section-number">4</span> Clinician datasets</a><ul>
<li><a href="#dataset-format"><span class="toc-section-number">4.1</span> Dataset format</a></li>
<li><a href="#data-normalisation"><span class="toc-section-number">4.2</span> Data normalisation</a><ul>
<li><a href="#gene-expression-data"><span class="toc-section-number">4.2.1</span> Gene expression data</a></li>
<li><a href="#methylation-data"><span class="toc-section-number">4.2.2</span> Methylation data</a></li>
</ul></li>
<li><a href="#data-transformation"><span class="toc-section-number">4.3</span> Data transformation</a><ul>
<li><a href="#gene-expression-data-1"><span class="toc-section-number">4.3.1</span> Gene expression data</a></li>
<li><a href="#methylation-data-1"><span class="toc-section-number">4.3.2</span> Methylation data</a></li>
</ul></li>
</ul></li>
<li><a href="#clinician-dataset-metadata"><span class="toc-section-number">5</span> Clinician dataset metadata</a></li>
</ul>
</div>

<div id="overview-scientific-and-clinical-context" class="section level1">
<h1><span class="header-section-number">1</span> Overview: Scientific and clinical context</h1>
<p>Each tumor composition is different from a patient to another: different type of cell populations might be present in various proportions</p>
<p><img src="fig/tumor_heterogeneity.png" alt="Cancer heterogeneity illustration. Each tumor is a mix of different cell populations in various proportions." />.</p>
<p>Each tumor is a mix of different cell populations in various proportions. Advanced microdissection techniques to isolate a population of interest from heterogeneous clinical tissue samples are not feasible in daily practice. An alternative is to rely on computational deconvolution methods that infer cell-type composition from mixed (=”bulk”) samples.</p>
<p>This variable composition might be associated to difference in treatment response as well as distinct prognosis as shown by our team and others (Blum et al 2019, Nature Communication). It is therefore a high priority to describe as best as possible this “intra tumor” heterogeneity to better predict the more appropriate clinical care for each patient and go even further in personalized medicine.</p>
<p><strong>The goal of the cometh web application</strong> is to quantify tumor heterogeneity within clinical cancer omic samples (how many cell types are present? In which proportion?).</p>
</div>
<div id="computational-methods" class="section level1">
<h1><span class="header-section-number">2</span> Computational methods</h1>
<p>Computational methods identify an <strong>estimate of the cell type proportion matrix</strong>, given the predicted number of <strong>K</strong> cell types.</p>
<p>This computational method consists in solving a “convolution” equation to estimate the matrix <span class="math inline">\(A\)</span> (here of cell type proportions) of size <span class="math inline">\(KxN\)</span>, with <span class="math inline">\(K\)</span> the putative number of cell populations and N the number of samples :</p>
<p><img src="fig/deconvolution.png" alt="Statistical model of cancer heterogeneity: “convolution” equation to estimate the matrix A of proportions." />.</p>
<p>where <span class="math inline">\(D\)</span> (size <span class="math inline">\(MxN\)</span>) represents the molecular data of bulk mixed samples, with <span class="math inline">\(M\)</span> the number of molecular features (genes for example) and <span class="math inline">\(T\)</span> (size <span class="math inline">\(MxK\)</span>) the molecular profiles of the cell types.</p>
<p>In our context, a deconvolution method will enable to deconvolve a molecular measurement of a tumor sample into its different cellular components and to quantify their proportions.</p>
<div id="the-different-types-of-deconvolution-methods" class="section level2">
<h2><span class="header-section-number">2.1</span> The different types of deconvolution methods</h2>
<p>Two types of deconvolution approaches have been proposed in the literature: either “reference-based” using profiles of known cell types and numbers of populations, or " reference-free" using, for example, non-negative matrix factorization (NMF) or independent component analysis (ICA). In addition, different types of molecular data can be extracted from biological material and used as a surrogate for in silico deconvolution. Most existing methods use either DNA methylation or gene expression data as a molecular indicator. The combination of different omics types to assess tumor heterogeneity has been poorly studied until now, which will be made possible by our innovative benchmarking platform.</p>
</div>
<div id="criteria-used-for-the-selection" class="section level2">
<h2><span class="header-section-number">2.2</span> Criteria used for the selection</h2>
<p>We first based our selection on the outcome of our <a href="https://tinyurl.com/hadaca2019">HADACA data challenge</a>, gathering a consortium of more than thirty international <a href="https://cancer-heterogeneity.github.io/data_challenges_HADACAconsortium.html">experts</a> in the field. We selected the five best methods among all the ones collectively discovered and implemented during the challenge, that predictde the real cell proportions with the highest accuracy (i.e. lowest Mean Absolute Error between the estimate and the ground truth).</p>
<p>These methods consist in different strategies :</p>
<ul>
<li>Omics type : transcriptome or methylome</li>
<li>Feature selections: variance based or ICA based</li>
<li>Deconvolution algorithms: Edec, Medecom, snmf, ICA</li>
</ul>
<p>In addition, we conducted an extensive review of the literature to select reference deconvolution methods based on the following criteria :</p>
<ul>
<li>Associated publication: method published in a high-impact journal and highly cited (relative to the year of publication)</li>
<li>Year of publication: the most recent methods have been prioritized</li>
<li>Implementation: preference for the R language (R software is free and open access)</li>
</ul>
</div>
<div id="selection-of-13-reference-methods" class="section level2">
<h2><span class="header-section-number">2.3</span> Selection of 13 reference methods</h2>
<p>We ended up with 13 reference methods including both reference-free and reference-based methods and dealing with transcriptome or methylome data.</p>
<p><img src="fig/methods.png" alt="Table describing the characteristics of each deconvolution method." />.</p>
</div>
</div>
<div id="evaluation-of-methods-performances" class="section level1">
<h1><span class="header-section-number">3</span> Evaluation of methods performances</h1>
<div id="reference-datasets" class="section level2">
<h2><span class="header-section-number">3.1</span> Reference datasets</h2>
<p>Computational methods are evaluated on a set of reference benchmark datasets. We generated 8 benchmarks datasets enabling a comprehensive evaluation of the methods.</p>
<p><img src="fig/dataset.png" alt="Benchmark datasets" />.</p>
<p>We assume a reference dataset is composed of :</p>
<ul>
<li>a data_test <strong>D</strong> matrix composed of omic value for <span class="math inline">\(N\)</span> samples, for <span class="math inline">\(M\)</span> given features (genes, methylation probes…).</li>
<li>a data_solution <strong>A</strong> matrix composed of proportion value for <span class="math inline">\(N\)</span> samples, for <span class="math inline">\(K\)</span> cell types given features (genes, methylation probes…).</li>
</ul>
<p>For instance:</p>
<pre class="r"><code>dim(test_data_met)</code></pre>
<pre><code>## [1] 500   5</code></pre>
<pre class="r"><code>dim(test_data_rna)</code></pre>
<pre><code>## [1] 500   5</code></pre>
<pre class="r"><code>head(test_data_met)</code></pre>
<pre><code>##                 [,1]      [,2]      [,3]      [,4]      [,5]
## cg00000029 0.2824851 0.4656137 0.5446347 0.2820870 0.3567818
## cg00000109 0.7768669 0.8144932 0.8143146 0.6895472 0.7482526
## cg00000155 0.7316993 0.9585858 0.9717631 0.8410465 0.9583800
## cg00000158 0.7321741 0.8182967 0.8959911 0.8954138 0.8798266
## cg00000165 0.5179173 0.6241434 0.3472937 0.3072159 0.5405272
## cg00000221 0.7147090 0.7006120 0.6280123 0.9319660 0.4827374</code></pre>
<pre class="r"><code>head(test_data_rna)</code></pre>
<pre><code>##                [,1]       [,2]       [,3]        [,4]      [,5]
## TSPAN6   4.53412964 4.64524344 4.46117771 4.693632277 4.7787183
## TNMD     0.01756076 0.05738122 0.02154218 0.002280051 0.0692128
## DPM1     5.01491459 5.39509861 5.12859560 5.131644901 5.3757923
## SCYL3    4.27296046 3.64167946 3.77653212 3.856753286 3.9413723
## C1orf112 3.47073829 3.10029072 2.92953187 2.655649054 3.4656401
## FGR      1.22510287 1.94157661 2.31569565 1.395961242 0.5588283</code></pre>
<pre class="r"><code>dim(test_solution)</code></pre>
<pre><code>## [1] 5 5</code></pre>
<pre class="r"><code>head(test_solution)</code></pre>
<pre><code>##             [,1]        [,2]       [,3]        [,4]       [,5]
## [1,] 0.079102722 0.077857899 0.14571941 0.464455778 0.07166962
## [2,] 0.278741721 0.377048646 0.40214293 0.374365927 0.36659346
## [3,] 0.120638364 0.214034841 0.23976354 0.005731404 0.04557118
## [4,] 0.517661978 0.001631235 0.12301909 0.122623108 0.24905805
## [5,] 0.003855216 0.329427378 0.08935503 0.032823782 0.26710770</code></pre>
</div>
<div id="scoring-metrics" class="section level2">
<h2><span class="header-section-number">3.2</span> Scoring metrics</h2>
<p>The discriminating metrics are computed on the A matrix. The rank of each method correspond to the mean rank of each method based on several metric evaluating the accuracy of the methods :</p>
<ul>
<li><strong>mean absolute error</strong> between the estimate and the groundtruth.</li>
<li><strong>root mean square error</strong> between the estimate and the groundtruth.</li>
<li><strong>kendall correlation</strong> between the estimate and the groundtruth.</li>
<li><strong>spearman correlation</strong> between the estimate and the groundtruth.</li>
</ul>
</div>
<div id="benchmark-leaderboard" class="section level2">
<h2><span class="header-section-number">3.3</span> Benchmark leaderboard</h2>
<p>All evaluation of the methods are accessible through the benchmark leaderboard located on the codabench benchmark platform.</p>
</div>
</div>
<div id="clinician-datasets" class="section level1">
<h1><span class="header-section-number">4</span> Clinician datasets</h1>
<p>At this point, we assume you have a dataset on which you would like to perform tumor heterogeneity quantification. Before uploading to COMETH web app, you should first check your dataset format using the following guidelines.</p>
<div id="dataset-format" class="section level2">
<h2><span class="header-section-number">4.1</span> Dataset format</h2>
<p>Your dataset (corresponding to matrix <span class="math inline">\(D\)</span> in our model) should be in .csv format, with ‘;’ as separators.</p>
<pre class="r"><code>DT_file_name = &quot;data/dataset_clinician.csv&quot;
DT_clinician = read.table(DT_file_name, sep =&quot;;&quot;, header=TRUE, row.names = 1)</code></pre>
<p>You can test that your dataset is at the correct format by using the following R script</p>
<pre class="r"><code>print(sprintf(&quot;Your transcriptome data has %d samples (columns) of %d probes (rows)&quot;,
              ncol(DT_clinician),nrow(DT_clinician)))</code></pre>
<pre><code>## [1] &quot;Your transcriptome data has 12 samples (columns) of 23520 probes (rows)&quot;</code></pre>
<pre class="r"><code>head(DT_clinician[,1:min(4,ncol(DT_clinician))])</code></pre>
<pre><code>##          GSM1570043 GSM1570044 GSM1570045 GSM1570046
## A1BG       5.084187   5.352647   5.346210   5.430234
## A1BG-AS1   6.863607   6.819322   7.042982   7.240511
## A1CF       4.933038   5.190533   4.959159   4.860161
## A2M        6.778452   6.774715   7.446837   7.433180
## A2M-AS1    4.731402   5.027431   7.492709   6.547286
## A2ML1      3.727863   3.928135   3.958703   3.853147</code></pre>
</div>
<div id="data-normalisation" class="section level2">
<h2><span class="header-section-number">4.2</span> Data normalisation</h2>
<p>Your dataset should be normalized.</p>
<div id="gene-expression-data" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Gene expression data</h3>
<p>For gene expression data, we suggest the following normalization strategy:</p>
<ul>
<li>micro-array : rma from “affy”package in R</li>
<li>RNA-seq : DESeq, EdegR, transcripts per million (TPM)…</li>
</ul>
<p>You can test the normalization of your data using the following R script:</p>
</div>
<div id="methylation-data" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Methylation data</h3>
<p>For methylation array, we suggest the following normalization strategy:</p>
<ul>
<li>lumiMethylN from “lumi”package in R</li>
</ul>
<p>You can test the normalization of your data using the following R script:</p>
</div>
</div>
<div id="data-transformation" class="section level2">
<h2><span class="header-section-number">4.3</span> Data transformation</h2>
<p>Your dataset can be at linear scale of transformed. Please check the transformation (if any that was applied to your data)</p>
<div id="gene-expression-data-1" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Gene expression data</h3>
<p>Pseudo log transformation (log2+1) is a transformation often applied to expression data. You can test the transformation of your data using the following R script</p>
<pre class="r"><code>logcounts &lt;- log2(DT_clinician + 1 )
plot(DT_clinician[,1], DT_clinician[,2], cex=.1)</code></pre>
<p><img src="starting_kit_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
<pre class="r"><code>plot(logcounts[,1], logcounts[,2], cex=.1)</code></pre>
<p><img src="starting_kit_files/figure-html/unnamed-chunk-9-2.png" width="672" /></p>
<pre class="r"><code>pc &lt;- prcomp( t( DT_clinician ) )
plot(pc$x[,1], pc$x[,2])</code></pre>
<p><img src="starting_kit_files/figure-html/unnamed-chunk-9-3.png" width="672" /></p>
<pre class="r"><code>pc &lt;- prcomp( t( logcounts ) )
plot(pc$x[,1], pc$x[,2])</code></pre>
<p><img src="starting_kit_files/figure-html/unnamed-chunk-9-4.png" width="672" /></p>
</div>
<div id="methylation-data-1" class="section level3">
<h3><span class="header-section-number">4.3.2</span> Methylation data</h3>
<p>Methylation data are usually beta-values or m-values. You can test the transformation of your data using the following R script</p>
</div>
</div>
</div>
<div id="clinician-dataset-metadata" class="section level1">
<h1><span class="header-section-number">5</span> Clinician dataset metadata</h1>
<p>Along with your omic dataset, we ask you to specify the cancer type related to your data (to be chosen within the <a href="https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/tcga-study-abbreviations">TCGA references</a>) and to estimate the number of cell types <span class="math inline">\(K\)</span> of your data.</p>
<p>Please be prepared to fill this information on the COMETH web application.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
